src/Engines/nativeEngine.ts

```diff
@@ -1382,159 +1340,6 @@ export class NativeEngine extends Engine {
         return this._colorWrite;
     }
 
-    private applyStencil(): void {

```

Comment:
did you mean to remove these lines from the native engine in this PR? Is it related?

Reply:
Ah, since this is a draft I didn't bother removing the revert commit I needed to add to get things working with BabylonNative. I'll update it so the reverts are only on my local branch, though.